website/content/en/docs/reference/configuration/template-syntax.md

```diff
@@ -29,7 +29,7 @@ Notice that Vector allows direct field references as well as strftime specifiers
 
 The value of the key_prefix option would equal:
```

Comment:
Actually this should be:

```markdown
The value of the `key_prefix` option would equal:
```

Reply:
I mean, while we're at it...

Reply:
🤣

Reply:
Does `raw` look OK? it was the best option I dug up playing around (IMHO)